### ISSUE TITLE
update locations that rely on date hacks, once regex-replace func is merged into main in goskyr 

### ISSUE THREAD

**Issue by jakopako** [jakopako/croncert-config#186]
There are a couple of locations that rely on a bunch of hacks in goskyr to correctly parse the date. With the new regex-replace functionality (should be available from goskyr v0.3.5 thanks to @MarkJaroski ) the responsibilty of passing a correctly formatted date string to goskyr's date parsing func is shifted towards the person configuring a new concert location. Should websites contain invalidly formatted date strings this can be mitigated by replacing the respective string with a correctly formatted string.

Find all locations that rely on the goskyr hack and update their configs accordingly.
